pyomo/core/plugins/transform/__init__.py

```diff
@@ -9,15 +9,16 @@
 #  ___________________________________________________________________________
 
 import pyomo.core.plugins.transform.relax_integrality
-#import pyomo.core.plugins.transform.eliminate_fixed_vars
-#import pyomo.core.plugins.transform.standard_form
+# import pyomo.core.plugins.transform.eliminate_fixed_vars
```

Comment:
What is the reason for these commented out imports?

Reply:
I think at some point some of the transformations were deprecated and thus commented out. This PR doesn't change that fact, other than a cosmetic addition of a space after the `#`.

Reply:
OK, I see. In my view it looked like they were added and then commented out.

Reply:
I believe that @whart222 "deprecated" them by removing their registration and seeing if anyone complained.